Server/API/OrganizationManagementController.cs

```diff
@@ -89,6 +91,20 @@ public async Task<IActionResult> DeleteUser(string userID)
             return Ok("ok");
         }
 
+        [HttpGet("DeviceGroup")]
+        [ServiceFilter(typeof(ApiAuthorizationFilter))]
+        public IActionResult DeviceGroup()
+        {
+            if (User.Identity.IsAuthenticated &&
+                DataService.GetUserByNameWithOrg(User.Identity.Name).IsAdministrator)
+            {
+                return Ok(DataService.GetDeviceGroups(User.Identity.Name));
+            }
+            if (Request.Headers.TryGetValue("OrganizationID", out var orgID))
```

Comment:
Is there any authorization needed here, to ensure the user has access to the `OrganizationID` they specified?

Reply:
Er, I guess this is meant to be publicly accessible by the org id (since this check is only for when the User doesn't exist or isn't an admin), but I'm wondering why it _should_ be public

Reply:
The `ApiAuthorizationFilter` ensures that the current request is coming from either a user (via cookie authentication) or API access token.  The organization ID for that user/token is added to the headers.